src/content/apply/designer.md

```diff
@@ -9,6 +9,7 @@ qualities:
   - Can commit to 10 hours a week
 formLink: "http://eepurl.com/gyY5lz"
 role: designer
+description: "Sandbox is Northeastern’s student led software consultancy and aims to unleash the power of software for researchers and students. Our clients are scientists in a wide range of disciplines, and we engineer solutions to accelerate their research. We also develop in-house tools for the Northeastern community, such as searchneu.com and GraduateNU (a tool to help students build a plan of study)."
```

Comment:
not sure what the copy for the description will actually be, this was just a placeholder at the time